boms/legacy-ee/pom.xml

```diff
@@ -232,7 +188,7 @@
             </dependency>
             <dependency>
                 <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-weld-transactions-legacy-namespace</artifactId>
+                <artifactId>wildfly-weld-transactions</artifactId>
```

Comment:
I don't think these legacy boms should include the jakarta.* dependencies. The boms only exist to support building the old javax modules.

These 'get rid of source-transform' PRs should be about removing use of the legacy boms, so the only change to them would be removing dependency entries that the build no longer needs, like you do above with some of the jipijapa ones